python/ray/autoscaler/updater.py

```diff
@@ -74,35 +81,33 @@ def get_node_ip(self):
             return self.provider.external_ip(self.node_id)
 
     def run(self):
-        print("NodeUpdater: Updating {} to {}, logging to {}".format(
-            self.node_id, self.runtime_hash, self.output_name))
+        logger.info(  # Note that this does not log via child logger
```

Comment:
Why not `self.logger` here?

Reply:
Oh - I thought the handler would redirect all statements but I guess that's not true; I'll fix this.